docs/internals.rst

```diff
@@ -66,20 +66,25 @@ Commit message format
 
 Where ``<type>`` is a prefix, which tells Semantic Release what kind of changes you made in the commit:
 
--  ``feat`` - New functionality added
--  ``fix`` - Broken functionality fixed
+-  ``feat`` - New functionality added (results in _minor_ version bump)
+-  ``fix`` - Broken functionality fixed (results in _patch_ version bump)
+-  ``refactor`` - Changes in code, but no changes in behavior
 -  ``perf`` - Performance improved
+-  ``style`` - Changes in code formatting
+-  ``test`` - Changes in tests
 -  ``docs`` - Changes in documentation
 -  ``chore`` - Changes in package or repository configuration
--  ``refactor`` - Changes in code, but no changes in behavior
--  ``test`` - Changes in tests
 
 In the rare cases when your changes break backwards compatibility, the message must include ``BREAKING CHANGE:``, followed by an explanation. That will result in bumping the major version.
 
 -  See `existing commits <https://github.com/apiaryio/dredd/commits/master>`__ as a reference
 -  `Commitizen CLI <https://github.com/commitizen/cz-cli>`__ can help you to create correct commit messages
 -  ``npm run lint`` validates format of your messages
 
+.. note::
+
+    We use ``refactor`` together with `BREAKING CHANGE:` for changes in code which only remove features. There doesn't seem to be a better category for that use case.
```

Comment:
Could you please add a full commit message example? It can be helpful:

```diff
We use ``refactor`` together with `BREAKING CHANGE:` for changes in code which only remove features. There doesn't seem to be a better category for that use case.

+ refactor: example of commit message
+ BREAKING CHANGE: Updates library API A, B, C.
```